test/json/array.json

```diff
@@ -14,7 +14,7 @@
              "year":1937
           },
           {
-             "name":Les Demoiselles d'Avignon",
```

Comment:
Okay, sure. Maybe they could be named to `invalidArray.json` so that the intent is clear?